x/stake/types/msg.go

```diff
@@ -206,21 +206,21 @@ func NewMsgBeginRedelegate(delegatorAddr, validatorSrcAddr,
 
 //nolint
 func (msg MsgBeginRedelegate) Type() string { return MsgType }
-func (msg MsgBeginRedelegate) GetSigners() []sdk.Address {
-	return []sdk.Address{msg.DelegatorAddr}
+func (msg MsgBeginRedelegate) GetSigners() []sdk.AccAddress {
+	return []sdk.AccAddress{msg.DelegatorAddr}
 }
 
 // get the bytes for the message signer to sign on
 func (msg MsgBeginRedelegate) GetSignBytes() []byte {
 	b, err := MsgCdc.MarshalJSON(struct {
-		DelegatorAddr    string `json:"delegator_addr"`
-		ValidatorSrcAddr string `json:"validator_src_addr"`
-		ValidatorDstAddr string `json:"validator_dst_addr"`
-		SharesAmount     string `json:"shares"`
+		DelegatorAddr    sdk.AccAddress `json:"delegator_addr"`
+		ValidatorSrcAddr sdk.AccAddress `json:"validator_src_addr"`
+		ValidatorDstAddr sdk.AccAddress `json:"validator_dst_addr"`
+		SharesAmount     string         `json:"shares"`
 	}{
-		DelegatorAddr:    sdk.MustBech32ifyAcc(msg.DelegatorAddr),
-		ValidatorSrcAddr: sdk.MustBech32ifyVal(msg.ValidatorSrcAddr),
-		ValidatorDstAddr: sdk.MustBech32ifyVal(msg.ValidatorDstAddr),
+		DelegatorAddr:    msg.DelegatorAddr,
+		ValidatorSrcAddr: msg.ValidatorSrcAddr.Bytes(),
```

Comment:
Why `.Bytes()`? Can't this just use the standard `MsgCdc.MarshalJSON` with no extra struct now?